### DISCUSSION TITLE
Why to disable compile vulkan on windows

### DISCUSSION THREAD

**Discussion by zhouhang95** [gfx-rs/wgpu#2806] [Q&A] [answered]
I only want to use dx12 on win, how to disable compile vulkan.

**Comment by cwfitzgerald** [comment, marked as answer]
We currently do not allow you to remove backends at runtime, only add rarely-supported ones. We might in some potential future version, but not now.

**Reply by zhouhang95** [reply]
Not at runtime, at compile.

**Reply by zhouhang95** [reply]
ash make bin size big, on windows only using DX12 is ok.

**Reply by grovesNL** [reply]
@zhouhang95 I think @cwfitzgerald meant to say "compile-time" instead of "runtime" in that comment. There is no way to remove backends at compile-time right now.
